diff --git a/urpm-repoclosure.pl b/urpm-repoclosure.pl
index e4d06ed..a1bc1c2 100755
--- a/urpm-repoclosure.pl
+++ b/urpm-repoclosure.pl
@@ -787,6 +787,7 @@ sub downloadFile($$)
     
     my $DownloadTo = $TMP_DIR."/extract/".get_filename($Url);
     $DownloadTo=~s/\.cz/\.xz/g; # cz == xz
+    $DownloadTo=~s/\.lzma/\.xz/g; # cz == xz
     my $Dir = get_dirname($DownloadTo);
     mkdir($Dir);
     system("wget -U '' --no-check-certificate \"$Url\" --connect-timeout=5 --tries=1 --output-document=\"$DownloadTo\" >/dev/null 2>&1");
